editor/plugins/canvas_item_editor_plugin.h

```diff
@@ -112,7 +112,11 @@ class CanvasItemEditor : public VBoxContainer {
 		SNAP_RELATIVE,
 		SNAP_CONFIGURE,
 		SNAP_USE_PIXEL,
-		SHOW_GRID,
+		GRID_VISIBILITY_MENU,
```

Comment:
Please make related enums values stating with the same prefix, it is a lot easier to maintain.